libs/libpahomqtt/Makefile

```diff
@@ -0,0 +1,40 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libpahomqtt
+PKG_VERSION:=1.3.9
+PKG_RELEASE:=1
+PKG_SOURCE_PROTO:=git
+
+PKG_SOURCE_URL:=git://github.com/eclipse/paho.mqtt.c.git
+PKG_SOURCE_VERSION:=3b7ae6348bc917d42c04efa962e4868c09bbde9f
+PKG_MIRROR_HASH:=
+
+PKG_MAINTAINER:=Nian Bohung <n0404.n0404@gmail.com>
+PKG_LICENSE:=Eclipse Public License 2.0
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/libpahomqtt
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=Eclipse paho mqtt c library
+  URL:=git://github.com/eclipse/paho.mqtt.c.git
```

Comment:
```suggestion
  URL:=https://github.com/eclipse/paho.mqtt.c
```

Reply:
I hope to get the source code over git protocol.

Reply:
This is not for dlwonading a source code. This is the URL for the project. It could be their GitHub site, documentation, etc.

Reply:
I have already fixed the problem.
Thank you.